### ISSUE TITLE
Automatic addition of this and final qualifier no longer works in 0.18

### ISSUE THREAD

**Issue by charlouze** [dubreuia/intellij-plugin-save-actions#96]
As my title says, this. and final qualifier are not added automatically to my code since I updated to 0.18...

Here are some informations about my intellij version:

IntelliJ IDEA 2017.2.1
Build #IU-172.3544.35, built on July 31, 2017
JRE: 1.8.0_152-release-915-b6 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.10.0-30-generic

**Comment by dubreuia** [comment]
@charlouze can you post your save-actions settings configuration please?

It seems some combination of quick fixes since #85 do not work

**Comment by charlouze** [comment]
@dubreuia Here are my settings:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="SaveActionSettings">
    <option name="actions">
      <set>
        <option value="activate" />
        <option value="organizeImports" />
        <option value="reformat" />
        <option value="rearrange" />
        <option value="localCanBeFinal" />
        <option value="fieldCanBeFinal" />
        <option value="explicitTypeCanBeDiamond" />
        <option value="unqualifiedFieldAccess" />
        <option value="suppressAnnotation" />
        <option value="finalPrivateMethod" />
        <option value="unnecessarySemicolon" />
        <option value="missingOverrideAnnotation" />
        <option value="useBlocks" />
      </set>
    </option>
  </component>
</project>
```

**Comment by SchulteMarkus** [comment]
**Workaround**

[Use v0.16 of this plugin](https://plugins.jetbrains.com/plugin/download?updateId=36254) (from ["Save Actions" at plugins.jetbrains.com/](https://plugins.jetbrains.com/plugin/7642-save-actions))

**Comment by dubreuia** [comment]
I'm back from vacations, I'll look into this one this week. It is a regression from 0.18

Thanks @SchulteMarkus, the workaround should work.

**Comment by markiewb** [comment]
> It seems some combination of quick fixes since #85 do not work
I guess it is not the culprit. 

Somewhere in https://github.com/dubreuia/intellij-plugin-save-actions/commit/966602c9bcc20fbe488cade17ace6bbbe57f1a8f must be the issue.

**Comment by markiewb** [comment]
@dubreuia: Please see my PR #99, which solves this issue.

**Comment by dubreuia** [comment]
Thank you so much for contributing to this plugin @markiewb, your fix seems to work. I've already pushed to master and tagged v0.19, I'm testing it this afternoon. 

I'll publish it to the Jetbrains plugin repo this afternoon if it's good.

**Comment by dubreuia** [comment]
Fixed v0.19
